app-generated-skeleton/AddedController.java

```diff
@@ -0,0 +1,14 @@
+package org.my.group;
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/hello-added")
+@Singleton
+public class AddedController {
+    @GET
+    public String sayHello() {
+        return "Hello added";
+    }
+}
```

Comment:
While Skeleton application started, except of SpringGreetingController.java class being created, there is also GreetingResource.java class. I don't see this class being used by any test. I suggest this class be used instead of creating a new file in the app-generated-skeleton folder unless there is some point not to do so.

Reply:
Don't take into account the first comment, I realized we need a new file.
What about copying an existing (e.x SpringGreetingController.java) with a different name, and then just change the endpoint?

Reply:
We need a new class to be added. `GreetingResource.java` is already present when the application starts, so it cannot be used to test live reload.

Reply:
To the copying approach: to me, having the new file prepared in the `app-generated-skeleton` folder seems more readable than generating and modifying a copy on the run. Are there any downsides of adding auxiliary files in this folder?

Reply:
I don't see any downside personally, wdyt @rsvoboda ?